docs/mine-hnt/data-only-guides/dragino-pg1301.mdx

```diff
@@ -0,0 +1,290 @@
+---
+id: dragino-pg1301
+title: Data Only Hotspot Setup Dragino PG1301 and Raspberry Pi
+sidebar_label: Dragino PG1301 + Raspberry Pi
+slug: /mine-hnt/data-only-guides/dragino-pg1301
+---
+
+# Data Only Hotspot Setup with Dragino PG1301 and Raspberry Pi
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+A Data Only Hotspot is a type of Hotspot that participates on the Helium Network
+by transferring data from the end devices (LoRaWAN nodes). The Data Only
+Hotspots are eligible for Network Data Transfer Rewards but not Proof of
+Coverage Rewards.
+
+This guide covers all the steps needed to set up a Data Only Hotspot using a
+Raspberry Pi and Dragino PG1301 hat.
+
+:::
+warning Data Only Hotspots cannot earn $HNT, at least for now, and after
```

Comment:
This needs to be like the following for all places you use these tags in the doc..

:::warning
text
:::